tips/tips-en.txt

```diff
@@ -109,3 +109,22 @@ GUI
 Hold SHIFT while scrolling with the mouse wheel to do page-wise scrolling in Tracker windows and drill-down context menus.
 https://www.haiku-os.org/docs/userguide/en/tracker.html
 %
+GUI
+You can quickly minimize an active window by using CTRL + ALT + M, and minimize all windows of that application by using CTRL + ALT + H.
+https://www.haiku-os.org/docs/userguide/en/keyboard-shortcuts.html
+%
+Miscellaneous
+Quickly switching to another workspace by pressing ALT + Fx (with Fx being the function key corresponding to workspace number X), add SHIFT to bring the active window with you.
+https://www.haiku-os.org/docs/userguide/en/keyboard-shortcuts.html
+%
+Preferences
+The Deskbar can be moved to all corners as well as the top or bottom of the screen, and also folded to a more compact layout by gripping and dragging the knobby area on the tray. 
```

Comment:
I think we need two sentences to be clear, and be specific about the knobby widget:

```suggestion
The Deskbar can be moved to all corners as well as the top or bottom of the screen by grabbing the knobbly widget beside the clock. It can be folded to a more compact layout by dragging the widget onto the tray. 
```